src/index.html

```diff
@@ -22,7 +22,7 @@
   DEALINGS IN THE SOFTWARE.
 -->
 
-<html>
+<html spellcheck="false">
```

Comment:
@redmunds It's not CEF specific afaik -- spell checking is a feature in most modern browsers, we just didn't see it before since our version of Chromium was so old :-/  I see the same behavior when running Brackets in-browser in Chrome, and presumably Safari & FF at least would be the same.